config/locales/de.yml

```diff
@@ -57,6 +66,14 @@ de:
               blank: 'Adresse ist erforderlich, bitte angeben'
             social_media_handle:
               blank: 'Social-Media-Handle ist erforderlich, bitte angeben'
+        interaction:
+          attributes:
+            date:
+              blank: 'Datum ist erforderlich, bitte angeben'
+            medium:
+              blank: 'Kommunikationsmittel ist erforderlich, bitte angeben'
```

Comment:
KOMMUNIKATIONSMITTEL!!!

Reply:
The german has to sound as german as possible i've decided

Reply:
😂